homeassistant/components/n26/sensor.py

```diff
@@ -0,0 +1,294 @@
+"""Support for N26 bank account sensors.
+
+The sensor allows to show
+* N26 account balance
+* Spaces (sub-accounts within an account) balance
+* bank card status
+"""
+from datetime import timedelta
+import logging
+
+from homeassistant.components.n26 import (
```

Comment:
```suggestion
from .. import (
```
Or extract them to `const.py`